inflection/tools/dictionary-parser/README.md

```diff
@@ -9,15 +9,16 @@ These tools generate files that describes the grammatical properties of words fr
 ## Usage for Wikidata
 
 1) Download a copy of Wikidata from https://dumps.wikimedia.org/wikidatawiki/entities/ (e.g. https://dumps.wikimedia.org/wikidatawiki/entities/20250115/wikidata-20250115-lexemes.json.bz2)
-2) Check what options were used for your language. They are at the end of the generated dictionary_XX.lst, look for "generated with options"
+2) Optionally decompress the file for improved reading performance.
+3) Check what options were used for your language. They are at the end of the generated dictionary_XX.lst, look for "generated with options"
    - Run `grep 'generated with options' ../../resources/org/unicode/inflection/dictionary/dictionary_XX.lst | cut -d':' -f2`
    - If the above command prints nothing, no additional options were used to generate the file, or it was generated with a different tool.
    - To see what options are available run `./ParseWikidata`
    - At minimum use the `--locale` option to specify the ISO-639 code for the language to extract.
-3) Run
+4) Run
 ```
-    ./ParseWikidata <THE_OPTIONS_FROM_STEP_2> \
+    ./ParseWikidata <THE_OPTIONS_FROM_STEP_3> \
     --inflections ../../resources/org/unicode/inflection/dictionary/inflectional_XX.xml \
     --dictionary ../../resources/org/unicode/inflection/dictionary/dictionary_XX.lst \
-    <wikidata-NNNNNNNN-lexemes.json.bz2>
+    <wikidata-NNNNNNNN-lexemes.json>
```

Comment:
We can keep .bz2 as an option in the documentation (one may not decompress to gain performance).

something like <wikidata-NNNNNN-lexemes.json[.bz2]>